index.html

```diff
@@ -6,33 +6,162 @@
+    <nav>
+      <a href="index.html">Soulemanou Babanou</a>
+      <img src="Resources/Images/Icon-Menu20.png" alt="main_icon">
+    </nav>
+     <!--the headline section-->
+    <section id="headline">
+      <h1>Hey There.<span>I’m Babanou</span></h1>
+      <h2>I’am a Software Developer</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+        <ul>
+          <li><a href="#"><img src="Resources/Images/Vector1.png" alt="Github"></a></li>
+          <li><a href="#"><img src="Resources/Images/Icon-Linkedin.png" alt="Linkin"></a></li>
+          <li><a href="#"><img src="Resources/Images/Icon-Angelist.png" alt="Angelist"></a></li>
+          <li><a href="#"><img src="Resources/Images/Icon-Twitter.png" alt="Twitter"></a></li>
+          <li><a href="#"><img src="Resources/Images/Icon-Medium.png" alt="Medium"></a></li>
+        </ul>
+    </section>
+     <!--the work section-->
+    <section id="work">
+      <h2>My Recent Works</h2>
+      <img src="Resources/Images/ImgPlaceholder.png" alt="ImgPlaceholder">
+      <h2>Multi-Post Stories</h2>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul>
+        <li>Css</li>
+        <li>Html</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" name="See_Project">See Project</button>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h2>Profesional Art <span>Printing Data</span> </h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+       <!--work cart-->
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
```

Comment:
- [ ] For best practice, Kindly change the `<h1>` tag here to any of `<h2>` to `<h6>`. It's a good practice to have just one  `<h1>` tag on a webpage for the main title of the page. This will improve the SEO of your page. You can read more [here](https://www.semrush.com/blog/h1-tag/)